dll/cpl/desk/lang/zh-CN.rc

```diff
@@ -84,7 +85,7 @@ CAPTION "高级外观"
 FONT 9, "宋体"
 BEGIN
     CONTROL "", IDC_APPEARANCE_PREVIEW, "PreviewWndClass", WS_VISIBLE | WS_BORDER, 7, 7, 232, 120
-    LTEXT "If you select a theme other than ReactOS classic, it will override the following settings, except in some older programs.", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
+    LTEXT "如果您选择 ReactOS 经典 之外的其他主题，它会覆盖下面的设置，某些老旧的程序除外。", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
```

Comment:
```suggestion
    LTEXT "如果您选择 ReactOS 经典 之外的其他主题，它会覆盖下面的设置，除了在某些老旧的程序内之外。", IDC_ADVAPPEARANCE_DISCLAIMER, 7, 133, 232, 30, BS_MULTILINE
```

Reply:
I'd like to discuss this further. It still looks kinda weird